gitlab/files/gitlab.rb

```diff
@@ -11,7 +11,7 @@
 ##! URL on which GitLab will be reachable.
 ##! For more details on configuring external_url see:
 ##! https://docs.gitlab.com/omnibus/settings/configuration.html#configuring-the-external-url-for-gitlab
-external_url 'https://{{ server.server_name }}'
+external_url '{{ server.url }}'
```

Comment:
I think you should check if server.url is defined here to be backward compatible

```{%- if server.url is defined %}
external_url = '{{ server.url }}'
{%- else %}
external_url = 'https://{{ server.server_name }}'
{%- endif %}```